upgrade/upgrade-scripts/common.sh

```diff
@@ -15,6 +15,13 @@
 # limitations under the License.
 #
 
+#
+# set umask to 0022 so that all new directories will be created with correct
+# permissions. This is especially required for not-in-place upgrades. This will
+# override the umask set by mgmt stack.
```

Comment:
I'd recommend we slightly modify this comment, now that it's contained in a more generic location.. e.g. something like:
```
#
# The virtualization service uses a different umask than the default. Thus to account for the
# fact that these scripts may be running with a non-default umask, we explicitly change it back
# to the default value here. This helps ensure any files and directories generated by these scripts,
# will be created with the correct permissions, regardless of the umask of the calling process.
#
```

Reply:
That looks good.